guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-capsule-server.adoc

```diff
@@ -35,45 +35,42 @@ If you set a wildcard value `*` for the certificate's Common Name `CN =` in the
 If the command is successful, it returns two `{certs-generate}` commands, one of which you must use to generate the certificate archive file for your {SmartProxyServer}.
 ifdef::satellite[]
 +
-.Example output of `katello-certs-check`
+.To use them inside a NEW $CAPSULE, run this command:
```

Comment:
In our documentation we have the convention of using hostnames. Perhaps we should use `_new-{smartproxy-example-com}_` instead of `$CAPSULE`.

Alternatively you can define an attribute for it to remove the duplication. Like:
```asciidoc
ifdef::satellite[]
:smartproxy-fqdn-variable: $CAPSULE
endif::[]
ifndef::satellite[]
:smartproxy-fqdn-variable: $FOREMAN_PROXY
endif::[]
```

Reply:
I recommend going with host names such as https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L107. We expect users to adjust those values if they're in italic font.

Reply:
According to your comment, I preferred Ewoud's suggestion above. :+1: